integration-cli/docker_cli_port_test.go

```diff
@@ -256,19 +258,18 @@ func (s *DockerSuite) TestPortHostBinding(c *check.C) {
 
 	out, _ = dockerCmd(c, "port", firstID, "80")
 
-	if !assertPortList(c, out, []string{"0.0.0.0:9876"}) {
-		c.Error("Port list is not correct")
-	}
+	err := assertPortList(c, out, []string{"0.0.0.0:9876"})
+	// Port list is not correct
+	c.Assert(err, checker.IsNil, check.Commentf("%v", err))
 
 	dockerCmd(c, "run", "--net=host", "busybox",
 		"nc", "localhost", "9876")
 
 	dockerCmd(c, "rm", "-f", firstID)
 
-	if _, _, err := dockerCmdWithError("run", "--net=host", "busybox",
-		"nc", "localhost", "9876"); err == nil {
-		c.Error("Port is still bound after the Container is removed")
-	}
+	_, _, err = dockerCmdWithError("run", "--net=host", "busybox", "nc", "localhost", "9876")
+	// Port is still bound after the Container is removed
+	c.Assert(err, checker.NotNil)
```

Comment:
Here it would be helpful to get the output from the command in a `check.Commentf`